app/controllers/people_controller.rb

```diff
@@ -6,19 +6,19 @@ class PeopleController < ApplicationController
   def signin
     #This is the sign-in sheet, not anything about authentication
     @people = Person.active
-    @department = "Police"
+    @department = Department.where(name: "Police")&.first
     @page_title = "Sign-in"
     render :layout => "print_signin"
   end
 
   def orgchart
-    @people = Person.police.active
+    @people = Person.active.select{|person| person.department&.name == "Police"}
```

Comment:
Not for you to do, but we need to get these hard codes references out of the code base.